### PR TITLE
Lint deploy json

### PR DESCRIPTION
**Pull request by r4vi**
Add task to lint deploy.json to the webapp pipeline. This also parallelises any tasks that can be

### CODE REVIEW

**Comment by r4vi** [comment]
merging if fine but better if this is merged first: https://github.com/moj-analytical-services/deploy.json/pull/2/files

**Comment by r4vi** [comment]
@s-block if useful could probably add to the other pipeline types

**Review by andyhd — APPROVED** [review]
👍 Looks good to me

File: resource/webapp_pipeline.yaml [hunk 1]
```diff
@@ -2,48 +2,51 @@
 jobs:
 - name: deploy
   plan:
+  - aggregate:
```
Comment (xoen) [inline resource/webapp_pipeline.yaml:5]: Concurrency 🔀 😎

**Review by xoen — COMMENTED** [review]
FWIW looks good to me 👍

**Pull request merged**